docs/Maintainers.md

```diff
@@ -0,0 +1,50 @@
+Maintainers
+===========
+
+This document describes the Maintainers Committee of Open Enclave. By our
+liberal contribution policy outlined in our
+[governance model](GovernanceModel.md), maintainers are committers that are
+trusted to grant new committer rights, and grant new membership into the
+Maintainers Committee.
+
+When making decisions, the Maintainers Committee uses a "consensus seeking"
+process. This means that most decisions should be reached by consensus, but when
+that fails, the committee calls for a vote where the majority wins. This is to
+prevent obstructionism by removing the possibility of a one person veto.
+
+Maintainers Committee Members
+-----------------------------
+
+PENDING
+
+Responsibilities
+----------------
+
+The primary responsibility of the committee is to grant new committer rights
```

Comment:
We should probably be more explicit about what the criteria is considered by the committee for adding committers.

Reply:
Why? That's something the maintainers can discuss and agree upon when seeking consensus.

Reply:
It's important for establishing transparency and trust in the process. How does this not come across as a kangaroo court if all of the initial maintainers are going to be from Microsoft?

Reply:
The key is that (hopefully) they won't be, but that's still under private discussion. Committers should be added with the least resistance possible. From the article:

> All contributors who land a non-trivial contribution should be on-boarded in a timely manner, and added as a committer, and be given write access to the repository.

This is explicitly to avoid the common open source issue of a project closely guarding its write access and causing the project to fail due to lack of influx of new committers. Fortunately, everyone I've reached out to become an official maintainer seems on-board with this philosophy (by virtue of that being our team and company's philosophy).